batch7crud-roll9/src/main/java/com/lftechnology/batch7crud/services/StudentService.java

```diff
@@ -0,0 +1,37 @@
+package com.lftechnology.batch7crud.services;
+
+import com.lftechnology.batch7crud.dao.StudentDAO;
+import com.lftechnology.batch7crud.entity.Student;
+import com.lftechnology.batch7crud.exception.DataException;
+import java.util.List;
+
+/**
+ * Created by sanjay on 1/14/16.
+ */
+public class StudentService {
+  private StudentDAO stdDAO = new StudentDAO();
+
+  public void save(Student s) throws DataException {
```

Comment:
it is always a best practice to return the same object after save/edit so that you can use the saved if needed.